### ISSUE TITLE
Pytorch Demo: all trails failed!  "cmd.exe : python: can't open file 'model.py'"

### ISSUE THREAD

**Issue by xixilllll** [microsoft/nni#4805]
**Describe the issue**:
I'm a freshman. When I run the pytorch demo, **all trails failed**, 
![image](https://user-images.githubusercontent.com/40450302/165095043-3459833e-fd28-4968-8f03-4e216e55936e.png)
and the trail error says " cmd.exe : python: can't open file 'model.py': [Errno 2] No such file or directory", and stdout is empty
![image](https://user-images.githubusercontent.com/40450302/165094800-47af582c-561a-4fd2-9102-7a4087203d73.png)

**Environment**:
- NNI version: 2.6.1
- Training service (local|remote|pai|aml|etc): local
- Client OS: windows10
- Server OS (for remote mode only): 
- Python version: 3.8
- PyTorch/TensorFlow version: 1.10.2 cuda
- Is conda/virtualenv/venv used?: use conda
- Is running in Docker?: No

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - 
 - Search space:

**Log message**:
 - nnimanager.log: 
 - [2022-04-25 20:43:26] INFO (NNIDataStore) Datastore initialization done
[2022-04-25 20:43:26] INFO (RestServer) RestServer start
[2022-04-25 20:43:26] WARNING (NNITensorboardManager) Tensorboard may not installed, if you want to use tensorboard, please check if tensorboard installed.
[2022-04-25 20:43:26] INFO (RestServer) RestServer base port is 8080
[2022-04-25 20:43:26] INFO (main) Rest server listening on: http://0.0.0.0:8080
[2022-04-25 20:43:29] INFO (NNIManager) Starting experiment: zwyfcrl7
[2022-04-25 20:43:29] INFO (NNIManager) Setup training service...
[2022-04-25 20:43:30] INFO (LocalTrainingService) Construct local machine training service.
[2022-04-25 20:43:30] INFO (NNIManager) Setup tuner...
[2022-04-25 20:43:30] INFO (NNIManager) Change NNIManager status from: INITIALIZED to: RUNNING
[2022-04-25 20:43:30] INFO (NNIManager) Add event listeners
[2022-04-25 20:43:30] INFO (LocalTrainingService) Run local machine training service.
[2022-04-25 20:43:30] INFO (NNIManager) NNIManager received command from dispatcher: ID, 
[2022-04-25 20:43:30] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 0, "parameter_source": "algorithm", "parameters": {"features": 128, "lr": 0.0008303904360771582, "momentum": 0.6181306569552493}, "parameter_index": 0}
[2022-04-25 20:43:30] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 1, "parameter_source": "algorithm", "parameters": {"features": 512, "lr": 0.0003695263964422363, "momentum": 0.4149188862339417}, "parameter_index": 0}
[2022-04-25 20:43:35] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 0,
  hyperParameters: {
    value: '{"parameter_id": 0, "parameter_source": "algorithm", "parameters": {"features": 128, "lr": 0.0008303904360771582, "momentum": 0.6181306569552493}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:43:35] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 1,
  hyperParameters: {
    value: '{"parameter_id": 1, "parameter_source": "algorithm", "parameters": {"features": 512, "lr": 0.0003695263964422363, "momentum": 0.4149188862339417}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:43:45] INFO (NNIManager) Trial job bSGpz status changed from WAITING to FAILED
[2022-04-25 20:43:45] INFO (NNIManager) Trial job x7sMq status changed from WAITING to FAILED
[2022-04-25 20:43:45] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 2, "parameter_source": "algorithm", "parameters": {"features": 512, "lr": 0.001089900485927162, "momentum": 0.008550750115060124}, "parameter_index": 0}
[2022-04-25 20:43:45] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 3, "parameter_source": "algorithm", "parameters": {"features": 128, "lr": 0.0012066180002456614, "momentum": 0.3589205910925304}, "parameter_index": 0}
[2022-04-25 20:43:50] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 2,
  hyperParameters: {
    value: '{"parameter_id": 2, "parameter_source": "algorithm", "parameters": {"features": 512, "lr": 0.001089900485927162, "momentum": 0.008550750115060124}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:43:50] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 3,
  hyperParameters: {
    value: '{"parameter_id": 3, "parameter_source": "algorithm", "parameters": {"features": 128, "lr": 0.0012066180002456614, "momentum": 0.3589205910925304}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:43:55] INFO (NNIManager) Trial job CWUBB status changed from WAITING to FAILED
[2022-04-25 20:43:55] INFO (NNIManager) Trial job X19kT status changed from WAITING to FAILED
[2022-04-25 20:43:55] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 4, "parameter_source": "algorithm", "parameters": {"features": 512, "lr": 0.000891197701762958, "momentum": 0.9183177099002631}, "parameter_index": 0}
[2022-04-25 20:43:55] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 5, "parameter_source": "algorithm", "parameters": {"features": 256, "lr": 0.00017781986380464154, "momentum": 0.5223018577419882}, "parameter_index": 0}
[2022-04-25 20:44:00] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 4,
  hyperParameters: {
    value: '{"parameter_id": 4, "parameter_source": "algorithm", "parameters": {"features": 512, "lr": 0.000891197701762958, "momentum": 0.9183177099002631}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:44:00] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 5,
  hyperParameters: {
    value: '{"parameter_id": 5, "parameter_source": "algorithm", "parameters": {"features": 256, "lr": 0.00017781986380464154, "momentum": 0.5223018577419882}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:44:06] INFO (NNIManager) Trial job rsS3c status changed from WAITING to FAILED
[2022-04-25 20:44:06] INFO (NNIManager) Trial job Wj2oy status changed from WAITING to FAILED
[2022-04-25 20:44:06] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 6, "parameter_source": "algorithm", "parameters": {"features": 256, "lr": 0.0001790518130213302, "momentum": 0.358621599855963}, "parameter_index": 0}
[2022-04-25 20:44:06] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 7, "parameter_source": "algorithm", "parameters": {"features": 256, "lr": 0.0016370084611949665, "momentum": 0.03431602566387759}, "parameter_index": 0}
[2022-04-25 20:44:11] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 6,
  hyperParameters: {
    value: '{"parameter_id": 6, "parameter_source": "algorithm", "parameters": {"features": 256, "lr": 0.0001790518130213302, "momentum": 0.358621599855963}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:44:11] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 7,
  hyperParameters: {
    value: '{"parameter_id": 7, "parameter_source": "algorithm", "parameters": {"features": 256, "lr": 0.0016370084611949665, "momentum": 0.03431602566387759}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:44:16] INFO (NNIManager) Trial job hVzMt status changed from WAITING to FAILED
[2022-04-25 20:44:16] INFO (NNIManager) Trial job UHK5I status changed from WAITING to FAILED
[2022-04-25 20:44:16] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 8, "parameter_source": "algorithm", "parameters": {"features": 128, "lr": 0.002296909478824145, "momentum": 0.16713995462925002}, "parameter_index": 0}
[2022-04-25 20:44:16] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 9, "parameter_source": "algorithm", "parameters": {"features": 128, "lr": 0.000847715846557005, "momentum": 0.4711180433456926}, "parameter_index": 0}
[2022-04-25 20:44:21] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 8,
  hyperParameters: {
    value: '{"parameter_id": 8, "parameter_source": "algorithm", "parameters": {"features": 128, "lr": 0.002296909478824145, "momentum": 0.16713995462925002}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:44:21] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 9,
  hyperParameters: {
    value: '{"parameter_id": 9, "parameter_source": "algorithm", "parameters": {"features": 128, "lr": 0.000847715846557005, "momentum": 0.4711180433456926}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2022-04-25 20:44:26] INFO (NNIManager) Trial job ZQZzn status changed from WAITING to FAILED
[2022-04-25 20:44:26] INFO (NNIManager) Trial job R0nfO status changed from WAITING to FAILED
[2022-04-25 20:44:26] INFO (NNIManager) Change NNIManager status from: RUNNING to: NO_MORE_TRIAL
[2022-04-25 20:44:26] INFO (NNIManager) Change NNIManager status from: NO_MORE_TRIAL to: DONE
[2022-04-25 20:44:26] INFO (NNIManager) Experiment done.
[2022-04-25 20:44:32] INFO (NNIManager) Change NNIManager status from: DONE to: STOPPING
[2022-04-25 20:44:32] INFO (NNIManager) Stopping experiment, cleaning up ...
[2022-04-25 20:44:33] INFO (LocalTrainingService) Stopping local machine training service...
[2022-04-25 20:44:33] INFO (NNIManager) Change NNIManager status from: STOPPING to: STOPPED
[2022-04-25 20:44:33] INFO (NNIManager) Experiment stopped.
[2022-04-25 20:44:33] INFO (NNITensorboardManager) Forced stopping all tensorboard task.
[2022-04-25 20:44:33] INFO (NNITensorboardManager) All tensorboard task stopped.
[2022-04-25 20:44:33] INFO (NNITensorboardManager) Tensorboard manager stopped.
[2022-04-25 20:45:19] INFO (NNIDataStore) Datastore initialization done
[2022-04-25 20:45:19] INFO (RestServer) RestServer start
[2022-04-25 20:45:19] WARNING (NNITensorboardManager) Tensorboard may not installed, if you want to use tensorboard, please check if tensorboard installed.
[2022-04-25 20:45:19] INFO (RestServer) RestServer base port is 8080
[2022-04-25 20:45:19] INFO (main) Rest server listening on: http://0.0.0.0:8080
[2022-04-25 20:45:23] INFO (NNIManager) Resuming experiment: zwyfcrl7
[2022-04-25 20:45:23] INFO (NNIManager) Setup training service...
[2022-04-25 20:45:23] INFO (LocalTrainingService) Construct local machine training service.
[2022-04-25 20:45:23] INFO (NNIManager) Change NNIManager status from: INITIALIZED to: VIEWED
[2022-04-25 20:46:18] ERROR (NNIRestHandler) Error: File not found: C:\Users\88305\nni-experiments\zwyfcrl7\trials\bSGpz\trial.log
    at LocalTrainingService.getTrialFile (F:\software\anaconda3\lib\site-packages\nni_node\training_service\local\localTrainingService.js:146:19)
    at NNIManager.getTrialFile (F:\software\anaconda3\lib\site-packages\nni_node\core\nnimanager.js:351:37)
    at F:\software\anaconda3\lib\site-packages\nni_node\rest_server\restHandler.js:287:29
    at Layer.handle [as handle_request] (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\layer.js:95:5)
    at next (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\layer.js:95:5)
    at F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\index.js:281:22
    at param (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\index.js:360:14)
    at param (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\index.js:371:14)
[2022-04-25 21:05:14] ERROR (NNIRestHandler) Error: File not found: C:\Users\88305\nni-experiments\zwyfcrl7\trials\bSGpz\trial.log
    at LocalTrainingService.getTrialFile (F:\software\anaconda3\lib\site-packages\nni_node\training_service\local\localTrainingService.js:146:19)
    at NNIManager.getTrialFile (F:\software\anaconda3\lib\site-packages\nni_node\core\nnimanager.js:351:37)
    at F:\software\anaconda3\lib\site-packages\nni_node\rest_server\restHandler.js:287:29
    at Layer.handle [as handle_request] (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\layer.js:95:5)
    at next (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\layer.js:95:5)
    at F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\index.js:281:22
    at param (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\index.js:360:14)
    at param (F:\software\anaconda3\lib\site-packages\nni_node\node_modules\express\lib\router\index.js:371:14)

 - dispatcher.log:
 - [2022-04-25 20:43:25] INFO (nni.experiment/MainThread) Creating experiment, Experiment ID: [36mzwyfcrl7[0m
[2022-04-25 20:43:25] INFO (nni.experiment/MainThread) Starting web server...
[2022-04-25 20:43:27] INFO (nni.experiment/MainThread) Setting up...
[2022-04-25 20:43:30] INFO (nni.experiment/MainThread) Web UI URLs: [36mhttp://192.168.199.132:8080 http://169.254.226.180:8080 http://127.0.0.1:8080[0m
[2022-04-25 20:43:30] INFO (nni.tuner.tpe/MainThread) Using random seed 28781843
[2022-04-25 20:43:30] INFO (nni.runtime.msg_dispatcher_base/MainThread) Dispatcher started
[2022-04-25 20:44:30] INFO (nni.experiment/MainThread) Stopping experiment, please wait...
[2022-04-25 20:44:32] INFO (nni.runtime.msg_dispatcher_base/MainThread) Dispatcher exiting...
[2022-04-25 20:44:33] INFO (nni.runtime.msg_dispatcher_base/MainThread) Dispatcher terminiated
[2022-04-25 20:45:19] INFO (nni.experiment/MainThread) Creating experiment, Experiment ID: [36mzwyfcrl7[0m
[2022-04-25 20:45:19] INFO (nni.experiment/MainThread) Starting web server...
[2022-04-25 20:45:21] INFO (nni.experiment/MainThread) Setting up...
[2022-04-25 20:45:23] INFO (nni.experiment/MainThread) Web UI URLs: [36mhttp://192.168.199.132:8080 http://169.254.226.180:8080 http://127.0.0.1:8080[0m

 - nnictl stdout and stderr: 
 

**How to reproduce it?**:

**Comment by liuzhe-lz** [comment]
Which demo are you using?
If you are using the [latest tutorial](https://nni.readthedocs.io/en/stable/tutorials/hpo_quickstart_pytorch/main.html), you need to download [model.py](https://nni.readthedocs.io/en/stable/tutorials/hpo_quickstart_pytorch/model.html) and put it in working directory.

**Comment by xixilllll** [comment]
> Which demo are you using? If you are using the [latest tutorial](https://nni.readthedocs.io/en/stable/tutorials/hpo_quickstart_pytorch/main.html), please download [model.py](https://nni.readthedocs.io/en/stable/tutorials/hpo_quickstart_pytorch/model.html) and put it in working directory. Download link: https://raw.githubusercontent.com/microsoft/nni/master/examples/tutorials/hpo_quickstart_pytorch/model.py

Thank you！
The model.py is correct, but still not work.  
But I tried to establish a new project in another file, it works unexpectedly！
Solution : Name the folder in English!
